packages/headless/src/app/commerce-engine/commerce-engine.ts

```diff
@@ -22,7 +22,11 @@ import {cartReducer} from '../../features/commerce/context/cart/cart-slice';
 
 export type {CommerceEngineConfiguration};
 
-const commerceEngineReducers = {productListing: productListingV2Reducer, commerceContext: contextReducer, cart: cartReducer};
+const commerceEngineReducers = {
```

Comment:
Looks like I had formatting issues when merging the context & cart PR. Thanks for fixing it here! 😁  

I assumed the pre-commit hook would format this, did I miss something? A ` npm run lint:fix packages/headless` I assume?

Reply:
`"lint:fix": "npx eslint --fix .",` has embedded path in it - so you would have to run `npx eslint --fix` directly with the path to lint a specific package. Wish it could be done in a separate PR and merged so it doesn't show up in all open PRs that merge from master - it shows up on 2 of my PRs already 😅